src/app/dashboard/artist-corner/form.tsx

```
@@ -0,0 +1,406 @@
+    const defaultValues =
+        mode === 'create'
+            ? {
+                  name: '',
+                  description: '',
+                  price: '0',
+                  images: [],
+                  download: '',
+                  is_free: false
+              }
+            : {
+                  name: initialData!.name,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)